apps/server/src/shared/testing/user-role-permissions.ts

```diff
@@ -194,6 +194,7 @@ export const teacherPermissions = [
 	'START_MEETING',
 	'TASK_CARD_VIEW',
 	'TASK_CARD_EDIT',
+	'CONTEXT_TOOL_ADMIN',
```

Comment:
From a type safety standpoint it would be better to use the enum itself instead of the strings that are associated with it, like this:
```
export const userPermissions: Permission[] = [
	Permission.ACCOUNT_CREATE,
	Permission.CLASS_CREATE,
]
```
But I guess this is a problem for Cedric to solve.